imap_upstream.adoc

```diff
@@ -16,23 +16,29 @@ IMAP related configuration options are available:
 
 The available balancing algorithms are as follows:
 
-* ClientIpHash
+* AccountIdHash
 
-  Selects an IMAP server server based on a hash of the client IP address.
-  The hashing algorithm used is a simple modulus of the client's ip address as an integer with the number of
-   configured IMAP servers. This will always return the same 'index' for the same client ip address.
-
-   Note: Extra configuration is necessary if a NAT proxy is being utilized in the environment.
+  Selects an IMAP server server based on a hash of the account id of
```

Comment:
It looks like you're wrapping text here...

nit: I'm not sure what you used as your fill-column / textwidth, but I'd probably wrap at column 70 vs whatever you used (the word 'the') would have ended up on this line) - but I like that we're at least doing some wrapping :)

Reply:
I am using Emacs built-in "M-q Fill current paragraph (fill-paragraph). " Although I didn't do any configuration as far as line length goes.

Reply:
Weird, usually 70 is the default for emacs, but that text looks like it's wrapping before 70.  Not a huge deal.

Reply:
I checked it actually is breaking at 70 :).
Perhaps we should adjust the wrap column of the entire document at 80?
Will address this in ZCS-4109.